main/webapp/modules/core/scripts/views/data-table/column-header-ui.js

```diff
@@ -58,6 +59,39 @@ DataTableColumnHeaderUI.prototype.getColumn = function() {
   return this._column;
 };
 
+// global state for the resizing of column headers
+DataTableColumnHeaderUI.resizingState = {
+  dragging: false, // whether we are currently resizing any column
+  col: null, // the column being resized
+  columnName: null, // the name of the column being resized
+  originalWidth: 0, // the original width of the header when the dragging started
+  originalPosition: 0 // the original position of the cursor when the dragging started
+};
+
+// global event handlers to react to mouse moves during resizing
+$(function() {
+  $('body').on('mousemove', function(e) {
```

Comment:
Instead of always listening for mouse movements you could listen for mouse down, register a listener for mouse movement, and finally unregister it on mouse up. It should greatly bring down the number of times this runs and because it would only be active when the mouse is down there will be no performance penalty of having it on each element rather than globally.

Reply:
Done.